index.html

```diff
@@ -227,6 +231,29 @@ <h1 class="about-title">About <br>Myself</h1>
     </div>
 
   </section>
+
+  <footer class="contact-form">
+
+    <div class="form-bg"></div>
+
+    <div class="from-descript">
+
+      <h2 class="form-title">Contact me</h2>
+      <div class="contact-info">
+        <p>If you have an application you are interested in developing, a feature that you need build or a project that needs coding. I'd love to help with.</p>
+      </div>
+      
+      <form class="form-action" action="https://formspree.io/f/{form_id}" method="post">
+        <input name="text" class="lable" placeholder="Yeremias" type="email" maxlength="30" required>
+        <input name="Email" class="lable" placeholder="natajayanj14@gmail.com" id="email" type="email" required>
+        <textarea name="textarea" class="text-area" placeholder="Write your message here" maxlength="500"></textarea>
```

Comment:
- According to the project requirements, all input fields should have the `required` attribute, please include this for the `textarea` 👍 👍

Reply:
Fixed.